pennylane/interfaces/jax.py

```diff
@@ -463,7 +465,7 @@ def execute_wrapper_jvp(primals, tangents):
                 new_tapes,
                 tangents[0],
                 gradient_fn,
-                device.shot_vector,
+                None if new_device_interface else device.shot_vector,
```

Comment:
Shouldn't this have `tape.shots.shot_vector` if `new_device_interface==True`?

Reply:
it's a batch of tapes so there is not (necessarily) a single value to pass. I instead handle this in the code this is calling on a tape-by-tape basis (if we get there and shots is None, we use `tape.shots.shot_vector` there

Reply:
I've updated autograd now too, but there is still an assumption being made that if one tape has partitioned shots, they all do. what can ya do